ui/lib/sync/addon/components/secrets/page/overview.hbs

```diff
@@ -20,7 +20,129 @@
       </ul>
     </nav>
   </div>
-  {{! overview cards here }}
+  <Toolbar>
+    <ToolbarActions>
+      <ToolbarLink @route="secrets.destinations.create" @type="add" data-test-create-destination>
+        Create new destination
+      </ToolbarLink>
+    </ToolbarActions>
+  </Toolbar>
+
+  <OverviewCard @cardTitle="Secrets by destination" class="has-top-margin-l">
+    {{#if this.fetchAssociationsForDestinations.isRunning}}
+      <div data-test-sync-overview-loading>
+        <Icon @name="loading" @size="24" />
+        Loading destinations...
+      </div>
+    {{else if (not this.destinationMetrics)}}
+      <EmptyState
+        @title="Error fetching information"
+        @message="Ensure that the policy has access to read sync associations."
+      >
+        <DocLink @path="/vault/api-docs/system/secrets-sync#read-associations">
+          API reference
+        </DocLink>
+      </EmptyState>
+    {{else}}
+      <Hds::Table>
+        <:head as |H|>
+          <H.Tr>
+            <H.Th>Sync destination</H.Th>
+            <H.Th @align="right"># of secrets</H.Th>
+            <H.Th @align="right">Last updated</H.Th>
+            <H.Th @align="right">Actions</H.Th>
+          </H.Tr>
+        </:head>
+        <:body as |B|>
+          {{#each this.destinationMetrics as |data index|}}
+            <B.Tr data-test-overview-table-row>
+              <B.Td>
+                <Icon @name={{data.icon}} data-test-overview-table-icon={{index}} />
+                <span data-test-overview-table-name={{index}}>{{data.name}}</span>
+                {{#if data.status}}
+                  <Hds::Badge
+                    @text={{data.status}}
+                    @color={{if (eq data.status "All synced") "success"}}
+                    data-test-overview-table-badge={{index}}
+                  />
+                {{/if}}
+              </B.Td>
+              <B.Td @align="right" data-test-overview-table-total={{index}}>
+                {{data.associationCount}}
+              </B.Td>
+              <B.Td @align="right" data-test-overview-table-updated={{index}}>
+                {{#if data.lastUpdated}}
+                  {{date-format data.lastUpdated "MMMM do yyyy, h:mm:ss a"}}
+                {{else}}
+                  &mdash;
+                {{/if}}
```

Comment:
I noticed when adding the placeholder that it looked a bit off and realized that the column was right aligned in the designs so I updated it.